### DISCUSSION TITLE
Colonie Chunk loading

### DISCUSSION THREAD

**Discussion by ghost** [ldtteam/minecolonies#8550] [Q&A]
Hello, is it possible to keep the colonie running at all times, or at least when the owner/officer is in the dimension, no matter the server resource costs.
I often have to go explore or do something else and the colonie comes to an complete halt and freezes.

I´ve fiddled with the "forceloadcolony" and "colonyloadstrictness" settings, set it to "true" and "1" but even when i am in the colonie, some citizens won´t get out of bed in the morning, only when i´m 3-4 chunks close.
 
Thanks in advance :)

**Comment by Talyda** [comment]
Could you ask on discord please? It's much easier to help there. And a much better place to ask!
